install_files/securedrop-keyring/DEBIAN/control

```diff
@@ -6,5 +6,5 @@ Homepage: https://securedrop.org
 Package: securedrop-keyring
 Version: 0.1.0+0.4.3
 Architecture: amd64
-Depends: gnupg
+Depends: gnupg,securedrop-config
```

Comment:
**securedrop-config** should be a direct dependency of securedrop-ossec-* and securedrop-app-code